lms/djangoapps/grades/docs/decisions/0005-grade-freeze.rst

```diff
@@ -0,0 +1,30 @@
+Freezing Changes to Subsection and Course Grades
+------------------------------------------------
+
+Status
+======
+
+Accepted (circa Summer 2018)
+
+Context
+=======
+
+We would like to prevent subsection-level or course-level grades from being
+updated in a course that has ended, after some fixed period of time.  After that fixed
+period of time, we consider grades for that course "frozen" - they can no longer by changed
+by the system.  This is important for maintaining the integrity of grades and a learner's completion of a course,
+particularly in credit-bearing courses.
+
+Decisions
```

Comment:
It would be great if you can add a bullet point or 2 about the consequences of this stop-gap solution.  For example:

`Persistent grades will now be inconsistent with the values computed from input scores, course content, grading policy, etc if changes occur after the frozen date.`